data/json/mutations/mutations.json

```
@@ -5030,14 +5031,29 @@
     "description": "You can now extend and reabsorb sensory organs at will, and can therefore ignore most shocks that would overwhelm lesser beings.",
     "purifiable": false,
     "types": [ "PER" ],
-    "leads_to": [ "SLIMESPAWNER" ],
+    "leads_to": [ "SLIMESPAWNER", "SLIMEMIND" ],
     "prereqs": [ "PER_SLIME" ],
     "prereqs2": [ "AMORPHOUS" ],
     "threshreq": [ "THRESH_SLIME" ],
     "category": [ "SLIME" ],
     "flags": [ "EYE_MEMBRANE" ],
     "passive_mods": { "per_mod": 5 }
   },
+  {
+    "type": "mutation",
+    "id": "SLIMEMIND",
+    "name": { "str": "Slimewave" },
+    "points": 10,
+    "visibility": 2,
+    "ugliness": 10,
+    "description": "You can communicate with your extant mass without physical contact, using electrical impulses to absorb knowledge from nearby friendly slime.",
```

I'd say you don't know the mechanism, just that it happens.
```suggestion
    "description": "You can communicate with your extant mass without physical contact, absorbing knowledge from nearby friendly slime.",
```